awx/api/views/mixin.py

```diff
@@ -60,6 +61,21 @@ def destroy(self, request, *args, **kwargs):
         return Response(status=status.HTTP_204_NO_CONTENT)
 
 
+class OrganizationInstanceGroupMembershipMixin(object):
+    """
+    This mixin overloads attach/detach so that it calls Organization.save(),
+    to ensure instance group updates are persisted
+    """
+
+    def unattach(self, request, *args, **kwargs):
```

Comment:
maybe this needs to be applied to attach as well

Reply:
I tried it, but it doesn't solve the problem we have with attaching (which is that the order ends up random)